example/ML-KEM-1024-both.priv.txt

```diff
@@ -0,0 +1,113 @@
+SEQUENCE {
+  INTEGER { 0 }
+  SEQUENCE {
+    OBJECT_IDENTIFIER { 2.16.840.1.101.3.4.4.3 }
+  }
+  SEQUENCE {
```

Comment:
that doesn't seem right though... the tag here is an OCTET STRING, but it has a SEQUENCE in it

(the PEM file is fine, just this text decode is bad)

Reply:
I haven't updated the pretty print yet, thats why :)

Reply:
updated in [726ca33](https://github.com/lamps-wg/kyber-certificates/pull/100/commits/726ca33b904c2de541dd44617662fb9971a57ffa)